core/src/elo-connector/elo-connector.service.ts

```diff
@@ -238,7 +238,11 @@ export class EloConnectorService {
             if (!series.matchParent.scrimMeta) {
                 throw new Error(`Series with id=\`${seriesId}\` has no associated scrim, but markSeriesNCP was called with SeriesType.Scrim.`);
             }
-            winningTeam = await this.teamRepository.findOneOrFail(0); // TODO: ID for FA team
+            winningTeam = await this.teamRepository.findOneOrFail({
+                where: {
+                    id: 0,
+                },
+            }); // TODO: ID for FA team
```

Comment:
Does FA even have a team? To relate a player to a team, it would require a roster slot. For FA, WW, etc. you'd need a roster slot per player, no?

Reply:
Free agency is a team in MLE, in theory in Sprocket that will not exist, but we can revisit that later. I don't think I originally wrote this.

Reply:
Sounds good.